README.md

```diff
@@ -29,7 +29,7 @@ Just add `rusty-hook` as a dev dependency in your Cargo.toml file:
 
 ```toml
 [dev-dependencies]
-rusty-hook = "^0.12.0"
+rusty-hook = "^0.11.0"
```

Comment:
The PR title mentions v0.11.2 not being available either, but that doesn't match with what I'm seeing. Could expand on what you mean when you say 0.11.2 doesn't exist?

To the best of my knowledge 0.11.2 does exist and is the most recent version available on crates, so if we're going to update this I'd prefer to ahead and set it to the latest:

```suggestion
rusty-hook = "^0.11.2"
```

Reply:
thank you for the at length explanation! 

sorry, I forgot a comma. I meant 0.11.2 is the latest version and 0.12 doesn't exist. 

also, when I'm trying to combine 2 cargo commands, let's say:
pre-commit = ["cargo check", "cargo fmt -- --check"], I get an error. It's the same for pre-push as well.

Reply:
> also, when I'm trying to combine 2 cargo commands, let's say:
pre-commit = ["cargo check", "cargo clippy"], I get an error

Correct, though that's a different matter than the topic of this PR.

The ability to use an array was added as part of the unreleased v0.12.0, though I feel like the documentation could really use an update to better reflect this

https://github.com/swellaby/rusty-hook#hooks

(we should make it clear that standard `&&` chaining always works regardless of version, the array format is only in v0.12.0+, and probably make that clear in the example above too)